lib/commands/screenshot.js

```diff
@@ -1,6 +1,16 @@
+import logger from '../logger';
+
 let commands = {};
 
 commands.getScreenshot = async function () {
+  if (this.mjpegStream) {
+    const data = this.mjpegStream.lastChunkBase64;
+    if (data) {
+      return data;
```

Comment:
I still don't think that it is a good idea to break the spec this way and send images in jpeg format instead of png

Reply:
we could probably use jimp for this purpose